content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -270,6 +271,18 @@ to the IP of the your WSL instance (`ip addr show dev eth0`) (For example,
 installed, follow the official 
 [Kubernetes on Windows instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-windows).
 
+## k3ai [*Alhpa*]
+
+K3ai is a lightweight infrastructure in a box specifically built to install and configure AI tools and platforms to quickly experiment over laptops and edge devices.
+
+Is main goal is to provide the most simple way to install Kubeflow pipelines without having to worry about setting up requirements and dependencies.
+To install Kubeflow pipelines with k3ai all is needed is to run the following command:
+
+```SHELL
+curl -sfL https://github.com/kf5i/k3ai/releases/latest/download/install | bash -
+```
+ For more installation options please read the guide at: [https://kf5ai.gitbook.io/k3ai/](https://kf5ai.gitbook.io/k3ai/)
```

Comment:
```suggestion

For more information about K3ai, refer to the [official documentation](https://kf5ai.gitbook.io/k3ai/).
```

- A blank line after the end of the code/command line block.
- Although I think it's OK to use "read", for better a11y maybe it's best to use something else, like "refer to".
- Re: "please", check out this [section](https://developers.google.com/style/tone#politeness-and-use-of-please) in the Google docs style guide.
- I noticed there's currently only one way of installing K3ai - correct me if I'm wrong. So, maybe we can change "for more installation options" to "for more information about K3ai". WDYT? It sort of covers both "more installation options" and "more details".
- Add a full stop at the end of the sentence.